Podfile

```diff
@@ -27,7 +27,7 @@ pod 'HockeySDK', '~>3.6.0'
 pod 'Helpshift', '~>4.8.0'
 pod 'CTAssetsPickerController', '~> 2.7.0'
 pod 'WordPress-iOS-Shared', '0.2.2'
-pod 'WordPress-iOS-Editor', :git => 'git://github.com/wordpress-mobile/WordPress-Editor-iOS', :commit => 'bd67b336dd1ca5bc2c5a277fa08f3dac017adb17'
+pod 'WordPress-iOS-Editor', :git => 'git://github.com/wordpress-mobile/WordPress-Editor-iOS', :commit => '6882a21fc71d6c02efef277a5178f7c1f82121a6'
```

Comment:
Totes! That one didn't exist when I made the PR. Now that the pod is updated we can point to the latest and greatest.